code/__DEFINES/pipes.dm

```diff
@@ -21,8 +21,6 @@
 #define PIPE_TVALVE					18
 #define PIPE_MANIFOLD4W				19
 #define PIPE_CAP					20
-#define PIPE_OMNI_MIXER				21
-#define PIPE_OMNI_FILTER			22
 #define PIPE_UNIVERSAL				23
```

Comment:
My reason for not updating the numbers further down (changing 23 to 21 and vice versa) is I am worried that a lot of atmos devices will break, and I want this to not break things.

Reply:
Would it maybe make sense to put a comment or something here explaining this?